lib/streamlit/elements/widgets/data_editor.py

```diff
@@ -796,7 +796,7 @@ def data_editor(
 
         # stlite: Don't use Arrow
         # arrow_bytes = type_util.pyarrow_table_to_bytes(arrow_table)
-        df_bytes = type_util.data_frame_to_bytes(data_df)
+        arrow_bytes = type_util.data_frame_to_bytes(data_df)
```

Comment:
nit: if we reuse the `arrow_bytes` name here, we can slightly lower the required patches